_includes/implementation-card.liquid

```diff
@@ -0,0 +1,24 @@
+<div class="card">
+  <div class="content">
+  <div class="header"><h3>{% if language %}{{ language | split: ',' | join: ' / ' }}{% else %}{{ applicationCategory }}{% endif %}</h3></div>
+    <div class="description">
+      <div class="ui selection list">
+        {% if language %}
```

Comment:
Something in here is being too aggressive and including JavaScript entries in the Java section.